docker-compose-run.sh

```diff
@@ -20,12 +22,50 @@ mkdir -p $TMP_SUBMISSION_MOUNT
 cp -r $EXERCISE_MOUNT $(dirname $TMP_EXERCISE_MOUNT)
 cp -r $SUBMISSION_MOUNT $(dirname $TMP_SUBMISSION_MOUNT)
 
+# use Python to parse EXERCISE_JSON
+# function: prints a value from the EXERCISE_JSON
+# param $1: the key whose value should be returned
+parse_val_from_exercise_json () {
+  local parsejson="import json; print(json.loads('$EXERCISE_JSON').get('$1', ''))"
+  echo $(python3 -c "$parsejson")
+}
+
+# in personalized exercises, we need the file path to the correct instance
+PERSONALIZED_MOUNT=$(parse_val_from_exercise_json personalized_exercise)
+# personal directory is an additional feature in the personalization, but it might
+# be removed in the future since it is hacky and not really needed
+PERSONAL_DIR_MOUNT=$(parse_val_from_exercise_json personal_directory)
+
+# normal exercises do not use the personalized mount
+PERSONALIZED_ARG=''
+if [ -n "$PERSONALIZED_MOUNT" ]; then
+  TMP_PERSONALIZED_MOUNT=$TMP/_personalized/${PERSONALIZED_MOUNT##/srv/data/mgrader_ex-meta/}
```

Comment:
$GRADER_PERSONALIZED_CONTENT_PATH  contains the last part (path). Should be available in the script.